content/pt-br/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Agentes de execução do Contêiner
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+{{< note >}}
+Dockershim foi removido do projeto Kubernetes desde a versão 1.24. Para mais detalhes leia [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq).
+{{< /note >}}
+
+Você precisa instalar um {{< glossary_tooltip text="agente de execução do contêiner" length="all"  term_id="container-runtime" >}} em cada nó do cluster para que os pods possam ser executados. Esta página descreve as tarefas e o que está envolvido para configurar os nós. 
+
+O Kubernetes 1.24 requer um tempo de execução que esteja em conformidade com {{< glossary_tooltip term_id="cri" length="all" text="Container Runtime Interface">}} (CRI).
```

Comment:
Here: `requer um tempo de execução` maybe you can use the original word `runtime`, will be better to understand, what you think?

Reply:
So I replace with `requer um runtime`?

Reply:
Instead of "runtime" or "tempo de execução", should we maintain "agente de execução"? That's what we have standardized in our dictionary for Portuguese at this time.

@edsoncelio do you agree with "agente de execução"?